schema_based_on_diagram.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE patients(
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR(255) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_histories(
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    admitted_at TIMESTAMP NOT NULL,
+    patient_id INT NOT NULL,
+    status VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_medical_histories FOREIGN KEY(id) REFERENCES patients(id)
+);
+
+
+CREATE TABLE treatments(
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    type VARCHAR(255) NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_medical_histories FOREIGN KEY(id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE invoices(
+    id INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+    total_amount DECIMAL(8, 2) NOT NULL,
+    generated_at TIMESTAMP NOT NULL,
+    payed_at TIMESTAMP NOT NULL,
+    medical_history_id INT NOT NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE invoices(
+    id INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+    total_amount DECIMAL(8, 2) NOT NULL,
+    generated_at TIMESTAMP NOT NULL,
+    payed_at TIMESTAMP NOT NULL,
+    medical_history_id INT NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_medical_histories FOREIGN KEY(Id) REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] You currently have two tables named ```invoices``` and with similar content. Kindly adjust this

Reply:
ok im going to delete that thank you @Ghiftee